run.py

```diff
@@ -3,4 +3,4 @@
 from app.application import app
 
 DEV_PORT = os.getenv('DEV_PORT', 5050)
-app.run(debug=True, host='0.0.0.0', port=int(DEV_PORT))
+app.run(debug=True, host='0.0.0.0', port=int(DEV_PORT))
```

Comment:
run.py is not used in cloud foundry, it is the scripts that devs use to start Secure Message. So it's ok hardcoded for now.